### PR TITLE
add workaround to block preinstall for known issue with Electron 4.0.x

### PR DESCRIPTION
**Pull request by shiftkey**
I'm a maintainer on [`keytar`](https://github.com/atom/node-keytar) and had to deal with the Electron 3/4 transition which had a known ABI issue that impacted native modules, discussed here https://github.com/lgeiger/node-abi/issues/54. We were able to workaround it by blocking clients from trying to download these packages, and I thought I'd upstream this fix to see if you were interested in making this available more broadly for anyone else who happens to stumble into it.

In a nutshell, this setup of `prebuild` packages should not be used if they were created:

 - `electron` runtime
 - a target between `4.0.0` and `4.0.3` (inclusive)
 - ABI 64 (Node 4 now uses 69 for it's ABI)

The patch to `node-abi` to support this new ABI is now available, so that future prebuild installs will avoid all this drama: https://github.com/lgeiger/node-abi/commit/9e4c1a63e63f8994cc2f5e3efee43785c9eb1c8f

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Released in 5.2.4.

@shiftkey Thanks! :heart:
